src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -420,15 +420,15 @@ public FeedbackSessionQuestionsBundle getFeedbackSessionQuestionsForStudent(
         List<FeedbackQuestionAttributes> questions = fqLogic.getFeedbackQuestionsForStudents(feedbackSessionName,
                 courseId);
 
-
+        Set<String> hiddenInstructorEmails = getHiddenInstrutorEmails(courseId);
```

Comment:
To check if there are questions with instructor recipient type, we need to iterate through the feedback questions. This will result in 2 for loops in the method and also checking the recipient type twice (once in `updateBundleAndRecipientListWithResponses` and once in `getHiddenInstructors`).